jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -1495,7 +1474,7 @@ void checkSecurity(Class<?> definingClass, Class<?> referenceClass, int modifier
 
 						/* third check */
 						if (definingClass.getClassLoader() != referenceClassLoader) {
-							if (!doesClassLoaderDescendFrom(definingClass.getClassLoader(), accessClass.getClassLoader())) {
+							if (!com.ibm.oti.util.Util.doesClassLoaderDescendFrom(definingClass.getClassLoader(), accessClass.getClassLoader())) {
```

Comment:
you can - `import static com.ibm.oti.util.Util.doesClassLoaderDescendFrom;`. then, you can directly call `doesClassLoaderDescendFrom`.